src/__tests__/byText.test.tsx

```diff
@@ -146,7 +148,8 @@ describe('findBy options deprecations', () => {
 });
 
 test.skip('getByText works properly with custom text component', () => {
-  function BoldText({ children }) {
+  type BoldTextProps = { children: React.ReactNode };
```

Comment:
How about extracting this to higher scope:

```
type ChildrenProps = { children: React.ReactNode }
```

And then reuse for all tests expecting `children`?